usaspending_api/agencies/management/commands/load_agencies.py

```diff
@@ -0,0 +1,270 @@
+import logging
+
+from collections import namedtuple
+from distutils.util import strtobool
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from pathlib import Path
+from psycopg2.extras import execute_values
+from psycopg2.sql import SQL
+from usaspending_api.common.csv_helpers import read_csv_file_as_list_of_dictionaries
+from usaspending_api.common.etl import ETLQueryFile, ETLTable, mixins
+from usaspending_api.common.helpers.sql_helpers import get_connection
+from usaspending_api.common.helpers.text_helpers import standardize_nullable_whitespace as prep
+from usaspending_api.common.helpers.timing_helpers import Timer
+from usaspending_api.references.constants import DOD_ARMED_FORCES_CGAC
+
+logger = logging.getLogger("console")
+
+Agency = namedtuple(
+    "Agency",
+    [
+        "row_number",
+        "cgac_agency_code",
+        "agency_name",
+        "agency_abbreviation",
+        "frec",
+        "frec_entity_description",
+        "frec_abbreviation",
+        "subtier_code",
+        "subtier_name",
+        "subtier_abbreviation",
+        "toptier_flag",
+        "is_frec",
+        "user_selectable",
+        "mission",
+        "website",
+        "congressional_justification",
+        "icon_filename",
+        "include_toptier_without_subtier",
+    ],
+)
+
+MAX_CHANGES = 75
```

Comment:
What determined this number as the threshold?

Reply:
Literally just made the number up.  I wanted us to be able to make tweaks without issue, but any errors introduced by Excel or any large scale changes I wanted us to catch before they introduced hours of additional run time to the nightly pipeline.

In my mind, the work flow is:

- If you make an intentional large scale change, we should probably manually run the loader.
- If you just tweaked a description or added a couple new agencies, it should be ok to run them in the pipeline.

Reply:
I understood the workflow. Perhaps the number could be lowered to... 20?

Reply:
So I kind of had in my head the number 25, but since changes could affect all three agency tables, I multipled it by 3 to get 75 to allow for double and triple counting.  Keep in mind that there are around 1,500 subtier agencies so 75 is really a drop in a bucket.

Want I should split the difference and go with 60 (3 * 20)?

Reply:
Nope, 75 is great. I was only thinking of the top tier agencies